doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -72,22 +72,23 @@ The network fabric shall comprise redundant layer 2 connectivity between hosts.
 The requirements on infrastructure hardware used to deploy and run the RI2 are as follows:
 
 1. One (1) physical or virtual server dedicated as a jump / test host
-1. Six (6) physical or virtual servers, serving as either master (controller) or compute/worker nodes
+1. Five (5) physical or virtual servers, serving as either master (controller) or compute/worker nodes
 
 The jump / test host allows to install and manage the operation of the RI2. For example, the host can be used to execute the installer software to deploy and configure the RI2 on the other physical or virtual servers. Once installed, the host may be used to run test tools or it may be used to generate test traffic intended for performance measurements. To support the latter, the server provided for the jump / test host shall meet similar hardware requirements as the master (controller) / compute hosts. If the jump host is used only for deployment or execution of functional tests after deployment, a VM with the resources as outlined below is sufficient.
 
 Each server shall meet the following minimum specifications:
 
 * CPU
-  *  2x x86_64 CPU sockets, providing 20 cores each, 40 simultaneous multi-threads (SMT), at 2.2 GHz
+  *  2x x86_64 CPU sockets, providing 20 cores each, 40 simultaneous multi-threads (SMT), at 2.2 GHz. If using VM's, the servers must be equipped with at least 16 vCPUs.
```

Comment:
I realise that these are issues that exist and not new but should we take this opportunity to clean-up? Thanks
- Are we asking for 2-sockets? But how many sockets populated with CPUs? 
- Is the 20 cores "each" for the server or each of the CPUs?
- the 40 SMT threads is as a result of the SMT enablement -- so does that and the 16 vCPUs part of server specifications or server configurations?